recipes-core/images/avnet-image-full.inc

```diff
@@ -110,6 +110,15 @@ IMAGE_INSTALL_append_minized-sbc = "\
         wpa-supplicant \
 "
 
+IMAGE_INSTALL_append_mz = "\
```

Comment:
If you rebase your changes on top of the new 2021.1 branch, you will see that we use the python webserver for minized, mz and pz = all zynq platforms.

So we can remove those packages from IMAGE_INSTALL_append_{minized/mz/pz}, and add them in IMAGE_INSTALL_append_zynq